30_ComputeShaderPathTracer/main.cpp

```diff
@@ -730,6 +730,71 @@ class ComputeShaderPathtracer final : public examples::SimpleWindowedApplication
 
 					ImGui::Image(SceneTextureIndeex, ImGui::GetContentRegionAvail());
 
+					// Nabla Imgui backend MDI buffer info
+					{
+						auto* streamingBuffer = m_ui.manager->getStreamingBuffer();
+						const size_t totalAllocatedSize = streamingBuffer->get_total_size();
+						const size_t isUse = streamingBuffer->max_size();
+
+						float freePercentage = 100.0f * (float)(totalAllocatedSize - isUse) / (float)totalAllocatedSize;
+						float allocatedPercentage = 1.0f - (float)(totalAllocatedSize - isUse) / (float)totalAllocatedSize;
+
+						ImVec2 barSize = ImVec2(400, 30);
+						float windowPadding = 10.0f;
+						float verticalPadding = ImGui::GetStyle().FramePadding.y;
+
+						ImGui::SetNextWindowSize(ImVec2(barSize.x + 2 * windowPadding, 110 + verticalPadding), ImGuiCond_Always);
+						ImGui::Begin("Nabla Imgui MDI Buffer Info", nullptr, ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoScrollbar);
+
```

Comment:
you don;t need this debug, just allocate 1 MB you'll never have a problem

Reply:
just to clarify @nipunG314 , if you pull latest stuff you can allocate MDI streaming buffer [yourself](https://github.com/Devsh-Graphics-Programming/Nabla/blob/ee3fafe3dd03772da56743cf2a0a239fc483eef6/include/nbl/ext/ImGui/ImGui.h#L47) now, also make sure to have proper access, memory, allocation & usage flags by checking [this](https://github.com/Devsh-Graphics-Programming/Nabla/blob/ee3fafe3dd03772da56743cf2a0a239fc483eef6/src/nbl/ext/ImGui/ImGui.cpp#L675)

Reply:
Done